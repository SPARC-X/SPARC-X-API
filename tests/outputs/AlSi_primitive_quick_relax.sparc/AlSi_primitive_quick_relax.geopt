:RELAXSTEP: 1
:E(Ha): -5.774163825437492E+00
:R(Bohr):
   0.000000000000000    0.000000000000000    0.000000000000000
   3.657724819665000    3.622889345192000    3.657724819665000
:F(Ha/Bohr):
   0.113139770628281    0.087368929971100    0.113139763970798
  -0.113139770628281   -0.087368929971100   -0.113139763970798
:CELL:   4.9264800451E+00   4.9264800451E+00   4.9264800451E+00
:VOLUME:   8.4546412886E+01
:LATVEC:
  0.0000000000E+00   5.0000000000E-01   5.0000000000E-01 
  5.0000000000E-01   0.0000000000E+00   5.0000000000E-01 
  5.0000000000E-01   5.0000000000E-01   0.0000000000E+00 
:STRESS:
 -6.5373541539E+02   1.5358155364E+00   5.1364935602E+00 
  1.5358155364E+00  -6.4458329707E+02   1.5364037537E+00 
  5.1364935602E+00   1.5364037537E+00  -6.5373541688E+02
:RELAXSTEP: 2
:E(Ha): -5.775437935275552E+00
:R(Bohr):
   0.002194196103526    0.001694404758356    0.002194195974413
   3.655530623561474    3.621194940433644    3.655530623690587
:F(Ha/Bohr):
   0.110411027039840    0.085419023529961    0.110411024664740
  -0.110411027039840   -0.085419023529961   -0.110411024664740
:CELL:   4.9264800451E+00   4.9264800451E+00   4.9264800451E+00
:VOLUME:   8.4546412886E+01
:LATVEC:
  0.0000000000E+00   5.0000000000E-01   5.0000000000E-01 
  5.0000000000E-01   0.0000000000E+00   5.0000000000E-01 
  5.0000000000E-01   5.0000000000E-01   0.0000000000E+00 
:STRESS:
 -6.5313614259E+02   9.0682890659E-01   4.3139086021E+00 
  9.0682890659E-01  -6.4446221862E+02   9.0731929019E-01 
  4.3139086021E+00   9.0731929019E-01  -6.5313614220E+02
:RELAXSTEP: 3
:E(Ha): -5.794556768421822E+00
:R(Bohr):
   0.047359563668326    0.036636400120512    0.047359562567641
   3.610365255996675    3.586252945071489    3.610365257097359
:F(Ha/Bohr):
   0.052498938225036    0.042849311571086    0.052498939943199
  -0.052498938225036   -0.042849311571086   -0.052498939943199
:CELL:   4.9264800451E+00   4.9264800451E+00   4.9264800451E+00
:VOLUME:   8.4546412886E+01
:LATVEC:
  0.0000000000E+00   5.0000000000E-01   5.0000000000E-01 
  5.0000000000E-01   0.0000000000E+00   5.0000000000E-01 
  5.0000000000E-01   5.0000000000E-01   0.0000000000E+00 
:STRESS:
 -6.4415620548E+02  -8.6396949567E+00  -8.0141975097E+00 
 -8.6396949567E+00  -6.4250622311E+02  -8.6415211701E+00 
 -8.0141975097E+00  -8.6415211701E+00  -6.4415620598E+02
:RELAXSTEP: 4
:E(Ha): -5.800046313085624E+00
:R(Bohr):
   0.088817234444247    0.070409417644916    0.088817234618174
   3.568907585220755    3.552479927547084    3.568907585046826
:F(Ha/Bohr):
  -0.002316182978578   -0.000896147882701   -0.002316181899298
   0.002316182978578    0.000896147882701    0.002316181899298
:CELL:   4.9264800451E+00   4.9264800451E+00   4.9264800451E+00
:VOLUME:   8.4546412886E+01
:LATVEC:
  0.0000000000E+00   5.0000000000E-01   5.0000000000E-01 
  5.0000000000E-01   0.0000000000E+00   5.0000000000E-01 
  5.0000000000E-01   5.0000000000E-01   0.0000000000E+00 
:STRESS:
 -6.4167808183E+02  -1.1488628417E+01  -1.1484645019E+01 
 -1.1488628417E+01  -6.4166849406E+02  -1.1490464116E+01 
 -1.1484645019E+01  -1.1490464116E+01  -6.4167808184E+02
:RELAXSTEP: 5
:E(Ha): -5.800057594119798E+00
:R(Bohr):
   0.087177352609414    0.069437155473653    0.087177353155337
   3.570547467055587    3.553452189718348    3.570547466509663
:F(Ha/Bohr):
  -0.000124428376470    0.000340548405068   -0.000124428470943
   0.000124428376470   -0.000340548405068    0.000124428470943
:CELL:   4.9264800451E+00   4.9264800451E+00   4.9264800451E+00
:VOLUME:   8.4546412886E+01
:LATVEC:
  0.0000000000E+00   5.0000000000E-01   5.0000000000E-01 
  5.0000000000E-01   0.0000000000E+00   5.0000000000E-01 
  5.0000000000E-01   5.0000000000E-01   0.0000000000E+00 
:STRESS:
 -6.4167181594E+02  -1.1491312956E+01  -1.1491173282E+01 
 -1.1491312956E+01  -6.4167099339E+02  -1.1493172604E+01 
 -1.1491173282E+01  -1.1493172604E+01  -6.4167181583E+02
