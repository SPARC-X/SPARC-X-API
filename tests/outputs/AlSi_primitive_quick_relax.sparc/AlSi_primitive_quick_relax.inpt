# nprocs: 8

# Test: CuSi7 #
LATVEC_SCALE: 6.9670948946E+00   6.9670948946E+00   6.9670948946E+00
LATVEC:
0.0 0.5 0.5
0.5 0.0 0.5
0.5 0.5 0.0
MESH_SPACING: 0.3
FD_ORDER: 12
BC: P P P
EXCHANGE_CORRELATION: GGA_PBE
TOL_SCF: 1e-6

KPOINT_GRID: 2 2 2

PRINT_FORCES: 1
PRINT_ATOMS: 1
CALC_STRESS: 1

RELAX_FLAG: 1
RELAX_METHOD: LBFGS

PRINT_RELAXOUT: 1
RESTART_FLAG: 0