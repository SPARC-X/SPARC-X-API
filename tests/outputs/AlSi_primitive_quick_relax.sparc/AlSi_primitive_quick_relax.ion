#=========================
# format of ion file
#=========================
# ATOM_TYPE:   <atom type name> 
# PSEUDO_POT:  <path/to/pseudopotential/>
# N_TYPE_ATOM: <num of atoms of this type>
# ATOMIC_MASS: <mass of atom of this type> #(optional, for MD only)
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX: #(optional)
# <xrelax> <yrelax> <zrelax>
# ...

# this is a comment

ATOM_TYPE: Al                              # atom type 
N_TYPE_ATOM: 1                              # number of atoms of this type
PSEUDO_POT: ../../../psps/13_Al_3_1.9_1.9_pbe_n_v1.0.psp8     # pseudopotential
COORD_FRAC:                                      # coordinates follows
    0.00 0.00 0.00

ATOM_TYPE: Si                               # atom type 
N_TYPE_ATOM: 1                              # number of atoms of this type
PSEUDO_POT: ../../../psps/14_Si_4_1.9_1.9_pbe_n_v1.0.psp8     # pseudopotential
COORD_FRAC:                                      # coordinates follows
    0.52 0.53 0.52


