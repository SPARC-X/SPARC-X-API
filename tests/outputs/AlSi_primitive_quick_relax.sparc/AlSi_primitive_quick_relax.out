***************************************************************************
*                       SPARC (version Feb 03, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun Feb  5 13:39:04 2023                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 6.9670948946 6.9670948946 6.9670948946 
LATVEC:
0 0.5 0.5 
0.5 0 0.5 
0.5 0.5 0 
FD_GRID: 17 17 17
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
KPOINT_SHIFT: 0.5 0.5 0.5
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 8
CHEB_DEGREE: 31
CHEFSI_BOUND_FLAG: 0
RELAX_FLAG: 1
RELAX_METHOD: LBFGS
RELAX_NITER: 300
L_HISTORY: 20
L_FINIT_STP: 0.005
L_MAXMOV: 0.2
L_AUTOSCALE: 1
L_LINEOPT: 1
L_ICURV: 1
CALC_STRESS: 1
TWTIME: 1E+09
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-09
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 8.40E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_VELS: 1
PRINT_RESTART: 1
PRINT_RESTART_FQ: 1
PRINT_ENERGY_DENSITY: 0
TOL_RELAX: 5.00E-04
PRINT_RELAXOUT: 1
OUTPUT_FILE: AlSi_primitive_quick_relax/temp_run/AlSi_primitive_quick_relax
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
0.000000000000000 3.483547447300000 3.483547447300000 
3.483547447300000 0.000000000000000 3.483547447300000 
3.483547447300000 3.483547447300000 0.000000000000000 
Volume: 8.4546412886E+01 (Bohr^3)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 4
NP_BAND_PARAL: 4
NP_DOMAIN_PARAL: 1 1 2
NP_DOMAIN_PHI_PARAL: 2 2 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing                       :  0.289793 (Bohr)
Number of symmetry adapted k-points:  4
Output printed to                  :  AlSi_primitive_quick_relax/temp_run/AlSi_primitive_quick_relax.out
Relax output printed to            :  AlSi_primitive_quick_relax/temp_run/AlSi_primitive_quick_relax.geopt
Total number of atom types         :  2
Total number of atoms              :  2
Total number of electrons          :  7
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  ../psps/13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Pseudocharge radii of atom type 1  :  8.69 8.69 8.69 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Si 4
Pseudopotential                    :  ../psps/14_Si_4_1.9_1.9_pbe_n_v1.0.psp8
Pseudocharge radii of atom type 2  :  8.69 8.69 8.69 (x, y, z dir)
Number of atoms of type 2          :  1
Estimated total memory usage       :  15.44 MB
Estimated memory per processor     :  329.45 kB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.8895143837E+00        1.271E-01        0.187
2            -2.8883634626E+00        9.291E-02        0.052
3            -2.8870907136E+00        7.331E-03        0.050
4            -2.8870819524E+00        1.588E-03        0.049
5            -2.8870819458E+00        7.338E-04        0.050
6            -2.8870819116E+00        3.100E-05        0.046
7            -2.8870819086E+00        1.741E-05        0.047
8            -2.8870819120E+00        1.330E-06        0.043
9            -2.8870819127E+00        7.860E-07        0.044
Total number of SCF: 9     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.8870819127E+00 (Ha/atom)
Total free energy                  : -5.7741638254E+00 (Ha)
Band structure energy              :  2.4070078967E+00 (Ha)
Exchange correlation energy        : -3.3642210981E+00 (Ha)
Self and correction energy         : -1.0315270074E+01 (Ha)
-Entropy*kb*T                      : -8.4184872923E-12 (Ha)
Fermi level                        :  6.7146492940E-01 (Ha)
RMS force                          :  1.8230343884E-01 (Ha/Bohr)
Maximum force                      :  1.8230343884E-01 (Ha/Bohr)
Time for force calculation         :  0.089 (sec)
Pressure                           :  6.5068470978E+02 (GPa)
Maximum stress                     :  6.5373541688E+02 (GPa)
Time for stress calculation        :  0.241 (sec)
Relax step time                    :  1.094 (sec)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.8877192084E+00        1.279E-03        0.057
2            -2.8877190697E+00        8.709E-04        0.050
3            -2.8877189699E+00        1.117E-04        0.046
4            -2.8877189680E+00        2.224E-05        0.048
5            -2.8877189682E+00        3.532E-06        0.044
6            -2.8877189676E+00        1.948E-07        0.043
Total number of SCF: 6     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.8877189676E+00 (Ha/atom)
Total free energy                  : -5.7754379353E+00 (Ha)
Band structure energy              :  2.4068113390E+00 (Ha)
Exchange correlation energy        : -3.3641525948E+00 (Ha)
Self and correction energy         : -1.0315273220E+01 (Ha)
-Entropy*kb*T                      : -2.3963589323E-11 (Ha)
Fermi level                        :  6.7226554644E-01 (Ha)
RMS force                          :  1.7798201831E-01 (Ha/Bohr)
Maximum force                      :  1.7798201831E-01 (Ha/Bohr)
Time for force calculation         :  0.088 (sec)
Pressure                           :  6.5024483447E+02 (GPa)
Maximum stress                     :  6.5313614259E+02 (GPa)
Time for stress calculation        :  0.239 (sec)
Relax step time                    :  0.664 (sec)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.8973879001E+00        2.673E-02        0.060
2            -2.8973238341E+00        1.811E-02        0.051
3            -2.8972788846E+00        2.333E-03        0.049
4            -2.8972784170E+00        4.318E-04        0.048
5            -2.8972783819E+00        7.278E-05        0.047
6            -2.8972783819E+00        3.740E-06        0.045
7            -2.8972783826E+00        2.257E-06        0.045
8            -2.8972783842E+00        2.233E-07        0.043
Total number of SCF: 8     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.8972783842E+00 (Ha/atom)
Total free energy                  : -5.7945567684E+00 (Ha)
Band structure energy              :  2.4039679682E+00 (Ha)
Exchange correlation energy        : -3.3631070549E+00 (Ha)
Self and correction energy         : -1.0315305740E+01 (Ha)
-Entropy*kb*T                      : -6.7110485457E-12 (Ha)
Fermi level                        :  6.7004689844E-01 (Ha)
RMS force                          :  8.5722463287E-02 (Ha/Bohr)
Maximum force                      :  8.5722463287E-02 (Ha/Bohr)
Time for force calculation         :  0.087 (sec)
Pressure                           :  6.4360621152E+02 (GPa)
Maximum stress                     :  6.4415620598E+02 (GPa)
Time for stress calculation        :  0.237 (sec)
Relax step time                    :  0.765 (sec)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9000214503E+00        2.182E-03        0.056
2            -2.9000229341E+00        1.538E-03        0.050
3            -2.9000231612E+00        2.344E-04        0.047
4            -2.9000231667E+00        8.342E-05        0.046
5            -2.9000231626E+00        1.198E-05        0.045
6            -2.9000231615E+00        2.731E-06        0.044
7            -2.9000231565E+00        5.295E-07        0.044
Total number of SCF: 7     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9000231565E+00 (Ha/atom)
Total free energy                  : -5.8000463131E+00 (Ha)
Band structure energy              :  2.4031935353E+00 (Ha)
Exchange correlation energy        : -3.3627971395E+00 (Ha)
Self and correction energy         : -1.0315309550E+01 (Ha)
-Entropy*kb*T                      : -8.4165728863E-12 (Ha)
Fermi level                        :  6.2770669624E-01 (Ha)
RMS force                          :  3.3959510021E-03 (Ha/Bohr)
Maximum force                      :  3.3959510021E-03 (Ha/Bohr)
Time for force calculation         :  0.087 (sec)
Pressure                           :  6.4167488591E+02 (GPa)
Maximum stress                     :  6.4167808184E+02 (GPa)
Time for stress calculation        :  0.238 (sec)
Relax step time                    :  0.710 (sec)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9000287525E+00        5.135E-04        0.068
2            -2.9000287693E+00        3.664E-04        0.048
3            -2.9000287861E+00        2.697E-05        0.045
4            -2.9000287914E+00        1.444E-05        0.045
5            -2.9000287915E+00        2.653E-06        0.045
6            -2.9000287971E+00        1.759E-07        0.041
Total number of SCF: 6     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9000287971E+00 (Ha/atom)
Total free energy                  : -5.8000575941E+00 (Ha)
Band structure energy              :  2.4031898209E+00 (Ha)
Exchange correlation energy        : -3.3627968290E+00 (Ha)
Self and correction energy         : -1.0315309550E+01 (Ha)
-Entropy*kb*T                      : -8.4348374147E-12 (Ha)
Fermi level                        :  6.2770483036E-01 (Ha)
RMS force                          :  3.8332503368E-04 (Ha/Bohr)
Maximum force                      :  3.8332503368E-04 (Ha/Bohr)
Time for force calculation         :  0.087 (sec)
Pressure                           :  6.4167154172E+02 (GPa)
Maximum stress                     :  6.4167181594E+02 (GPa)
Time for stress calculation        :  0.238 (sec)
Relax step time                    :  0.667 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  3.991 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
